.github/workflows/build-docs.yml

```diff
@@ -0,0 +1,51 @@
+on:
+  push:
+    branches:
+      - master
+
+name: Build docs
+
+jobs:
+  build:
+    name: Build docs
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+      - name: Install Python
+        uses: actions/setup-python@v1
+        with:
+          python-version: 3.7
```

Comment:
Just a question, what happens if python 3.7 is not shipped in ubuntu-lastest (which will be true a day or another)?

Reply:
I just want to know which indicator we are supposed to monitor...

Reply:
the build fails :-). And then we change the config. But it would stop early enough not to push a wrong version of the doc, if this is your concern.